services/gateway/shared/utils.js

```diff
@@ -58,7 +61,40 @@ const response = {
 	},
 };
 
+const requireAllJson = async apiName => {
+	const data = {
+		definitions: {},
+	};
+	const dir = path.resolve(__dirname, `../apis/${apiName}/swagger`);
+	const result = await fs.readdirSync(dir);
+	await BluebirdPromise.map(
+		result,
+		async fileName => {
+			if (fileName === 'definitions') {
+				const definitions = await fs.readdirSync(`${dir}/definitions`);
+				await BluebirdPromise.map(
+					definitions,
+					async definition => {
+						/* eslint-disable-next-line import/no-dynamic-require */
+						const content = require(`${dir}/definitions/${definition}`);
+						Object.assign(data.definitions, content);
+					},
+				);
+			} else {
+				/* eslint-disable-next-line import/no-dynamic-require */
+				const content = require(`${dir}/${fileName}`);
+				Object.assign(data, content);
+			}
+		},
+	);
+	return data;
+};
```

Comment:
It should be possible to make it completely synchronous.
Also, please consider running a `forEach` loop on the `result` array. This should eliminate the necessity to import `bluebird`.